libs/blocks/global-navigation/global-navigation.js

```diff
@@ -204,9 +204,30 @@ const closeOnClickOutside = (e) => {
   }
 };
 
-const getIetfLocale = (ietfLocale) => {
-  const nonStandardLocaleMap = { no_NO: 'nb_NO' };
-  return nonStandardLocaleMap[ietfLocale] || ietfLocale;
+const getUniversalNavLocale = (locale) => {
+  const prefix = locale.prefix.replace('/', '');
+  const { ietf } = locale;
+  const DEFAULTLANG = {
+    de_DE: ['at', 'ch_de', 'lu_de'],
+    en_US: ['ae_en', 'africa', 'au', 'be_en', 'ca', 'cis_en', 'cy_en', 'eg_en', 'gr_en', 'hk_en', 'id_en', 'ie', 'il_en', 'in', 'kw_en', 'langstore', 'lu_en', 'mena_en', 'mt', 'my_en', 'ng', 'nz', 'ph_en', 'qa_en', 'sa_en', 'sg', 'th_en', 'uk', 'vn_en', 'za'],
```

Comment:
This looks like it might have a line length larger than 100, ESLint might complain about it

Reply:
Fixed, added multiple lines in there 👍